main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Main.java

```diff
@@ -105,6 +105,13 @@ public static void main(String[] argv) {
     // Output
     exportReport(args.getOutputBase(), noticeContainer);
     final long endNanos = System.nanoTime();
+    if (!feedContainer.isParsedSuccessfully()) {
```

Comment:
Should we move this to the end of the method and invoke `System.exit()` as well with an nonzero error code?

https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/System.html#exit(int)

I believe this would ensure that the CI builds on the GitHub workflow show up as failed as well.

Reply:
No, please let's not do that. Right now, the Validator always exits with 0 and all errors are stored in the JSON files, even runtime exceptions. I agree to revisit that behaviour and have non-zero exit codes but let's have a separate discussion for that and a separate PR.

Reply:
Sounds good - let's discuss separately 👍

Reply:
Actually, this fits well into https://github.com/MobilityData/gtfs-validator/issues/734.